llmfoundry/models/layers/attention.py

```diff
@@ -432,7 +460,11 @@ def forward(
         if self.clip_qkv:
             qkv = qkv.clamp(min=-self.clip_qkv, max=self.clip_qkv)
 
-        query, key, value = qkv.chunk(3, dim=2)
+        query, key, value = qkv.split([
+            self.d_model, self.kv_n_heads * self.head_dim,
+            self.kv_n_heads * self.head_dim
+        ],
+                                      dim=2)
```

Comment:
```suggestion
        query, key, value = qkv.split(
        [
            self.d_model, 
            self.kv_n_heads * self.head_dim,
            self.kv_n_heads * self.head_dim,
        ],
        dim=2,
        )
```

Does this version still pass lint?
(the version you have just looks weird)